index.js

```diff
@@ -0,0 +1,84 @@
+//necessary page attributes 
+const playerName = document.querySelector('h2')
+const playerNickname = document.querySelector('em')
+const playerLikes = document.querySelector('p')
+const playerImg = document.querySelector('img')
+const goalUl = document.querySelector('#goals')
+const likeBtn = document.querySelector(".like-button")
+const vid = document.querySelector('#new-goal-form')
+let currentId = ""
+
+  
+fetch ("http://localhost:3000/players/1") 
+.then(resp => resp.json())
+.then(playerObj => {
+    playerImg.src = playerObj.photo
+    playerImg.alt = playerObj.name
+    playerName.textContent = playerObj.name
+    playerNickname.textContent = playerObj.nickname
+    playerLikes.textContent = `${playerObj.likes} Likes`
+    currentId = playerObj.id
+
+})
+
+fetch ("http://localhost:3000/goals")
+.then(resp => resp.json())
+.then(goalsObj => {
+    goalsObj.forEach(goal => {
+        renderGoal(goal)
+    })
+})
```

Comment:
Your rendering logic is right, but you shouldn't need to do a separate fetch request for the goals - they should be nested as an array in the playerObj in your other fetch request. This request isn't quite right, since you're requesting *all* the goals, not just the goals for Mo Salah.